frontend/src/components/layout/navigation/MobileNavigation.tsx

```diff
@@ -63,107 +71,154 @@ export const MobileNavigation = (props: Props) => {
       : styles["not-active"];
 
   return (
-    <nav
-      aria-label={l10n.getString("nav-menu-mobile")}
-      className={`${styles["mobile-menu"]}`}
-    >
-      {/* Below we have conditional rendering of menu items  */}
-      <ul
-        id={`${styles["mobile-menu"]}`}
-        className={`${styles["menu-item-list"]} ${toggleMenuStateClass}`}
+    <>
+      {/* Mask and Phone Header on Mobile */}
+      {isLoggedIn && isFlagActive(runtimeData.data, "phones") ? (
+        <div
+          className={`${styles["nav-mask-phone"]} ${styles["hidden-desktop"]}`}
+        >
+          {/* // Mask Btn */}
+          <Link href={homePath}>
+            <a
+              className={`${styles["nav-mask-phone-icon"]} ${
+                router.pathname === "/accounts/profile"
+                  ? styles["is-active"]
+                  : null
+              }`}
+            >
+              <MaskIcon width={25} height={25} alt="Go to Masks Dashboard" />
```

Comment:
Oh, one additional thing: this and the other `alt` are not localised. Also, given that we're calling this "phone masking" (#2392), I imagine it'd be confusing to refer to email masks as just "masks".

You might also want to consider adding a `title`, to help folks understand what the icons mean (I _think_ phones usually show them when you press-and-hold a link? And otherwise it's useful on desktop with small windows.)

Reply:
Alt and title attributes added.
<img width="348" alt="image" src="https://user-images.githubusercontent.com/13066134/186747046-9ce026c2-92cf-491c-9a42-60ab86ad2348.png">

Reply:
And specified masks = email mask + phone masks. Good call. My brain is still in "alias = mask" "phone =/= mask" territory.